systemtest/src/main/java/io/strimzi/systemtest/resources/specific/HelmResource.java

```diff
@@ -115,7 +115,8 @@ private void clusterOperator(long operationTimeout, long reconciliationInterval)
                 entry("resources.limits.cpu", LIMITS_CPU),
                 entry("logLevelOverride", Environment.STRIMZI_LOG_LEVEL),
                 entry("fullReconciliationIntervalMs", Long.toString(reconciliationInterval)),
-                entry("operationTimeoutMs", Long.toString(operationTimeout)))
+                entry("operationTimeoutMs", Long.toString(operationTimeout)),
+                entry("featureGates", Environment.STRIMZI_LOG_LEVEL))
```

Comment:
`Environment.STRIMZI_LOG_LEVEL` does not look to be the right env.

Reply:
Thanks, that's wrong copy/paste. Fixed.